common/remoteviewinterface.cpp

```diff
@@ -37,15 +37,112 @@ QT_BEGIN_NAMESPACE
 GAMMARAY_ENUM_STREAM_OPERATORS(RemoteViewInterface::RequestMode)
 QT_END_NAMESPACE
 
+QDataStream &operator<<(QDataStream &s, Qt::TouchPointStates states)
+{
+    return s << (int)states;
+}
+
+QDataStream &operator>>(QDataStream &s, Qt::TouchPointStates &states)
+{
+    int st;
+    s >> st;
+    states = Qt::TouchPointStates(st);
+    return s;
+}
+
+QDataStream &operator<<(QDataStream &s, QTouchEvent::TouchPoint::InfoFlags flags)
+{
+    return s << (int)flags;
+}
+
+QDataStream &operator>>(QDataStream &s, QTouchEvent::TouchPoint::InfoFlags &flags)
+{
+    int f;
+    s >> f;
+    flags = QTouchEvent::TouchPoint::InfoFlags(f);
+    return s;
+}
+
+QDataStream &operator<<(QDataStream &s, const QList<QTouchEvent::TouchPoint> &points)
+{
+    s << points.count();
+    for (const auto &p: points) {
```

Comment:
we still support MSVC 2010, which doesn't support range-based for... The CI should have caught that, right?